Makefile.emscripten

```diff
@@ -2,6 +2,7 @@ HAVE_STATIC_DUMMY ?= 0
 ifeq ($(TARGET),)
 ifeq ($(LIBRETRO),)
 TARGET := retroarch.js
+LIBRETRO = dummy
```

Comment:
`dummy`?

Reply:
This was necessary for the case where retroarch is built without any core during the build step in CI that makes the “precompiled retroarch”.  If there’s no value for the libretro variable, the emscripten command line is missing a value for the compiled module name and errors out.  So we need *some* name here and since dummy was already used in HAVE_STATIC_DUMMY I reused it here.